### DISCUSSION TITLE
bridge number - environment variable

### DISCUSSION THREAD

**Discussion by mikl0s** [tteck/Proxmox#605] [Ideas]
I would love to be able to export a vmbr interface variable instead of having to download the scripts and edit them to use vmbr1 instead of vmbr0 for generating contianers in lxc :)

If this is already possible I apologize but I don't see how :)

**Comment by tteck** [comment]
Use advanced settings

![Screenshot 2022-10-06 6 40 10 AM](https://user-images.githubusercontent.com/21342670/194293114-5ca24849-96da-4a1b-9564-6f7a169d3679.png)
![Screenshot 2022-10-06 6 46 40 AM](https://user-images.githubusercontent.com/21342670/194294316-15960d2f-9002-4f07-83d3-a6a0993ff45c.png)
